testsuite/smoke-test/src/operational_tooling.rs

```diff
@@ -341,20 +340,6 @@ async fn test_set_operator_and_add_new_validator() {
         .import_private_key(OWNER_KEY, validator_key)
         .unwrap();
 
-    // Verify no validator operator
```

Comment:
probably good to leave a note here of what the goal was of the test and not delete this